UDV-Core/src/Modules/ConsultDoc/examples/consultDocConfig.json

```diff
@@ -1,12 +1,12 @@
 {
     "type": "class",
     "server":{
-      "url":"http://rict.liris.cnrs.fr/APIVilo3D/APIExtendedDocument/web/",
-      "documentsRepository":"documentsDirectory/",
-      "getAll": "app_dev.php/getDocuments?",
-      "add":"app_dev.php/addDocument",
-      "delete":"",
-      "update":""
+      "url":"http://rict.liris.cnrs.fr:9095/",
+      "documentsRepository":"documents_repository/",
+      "getAll": "getDocuments?",
+      "add":"addDocument",
+      "delete":"deleteDocument",
+      "update":"editDocument"
```

Comment:
Do you need to specify add, delete and update here since it is not used in this demo ?

Reply:
I don't!